pyodide-test-runner/pyodide_test_runner/fixture.py

```diff
@@ -80,7 +82,7 @@ def selenium_common(
             f"Unknown runner or browser: {runner_type} / {request.param}"
         )
 
-    dist_dir = request.config.getoption("--dist-dir")
+    dist_dir = Path(os.getcwd(), request.config.getoption("--dist-dir"))
```

Comment:
--dist-dir could also be an absolute path in which case this will not work.

Reply:
I think it works even if `--dist-dir` is an absolute path:

```
>>> Path("/abs1", "/abs2")
PosixPath('/abs2')
```